appvalidator/specs/webapps.py

```diff
@@ -95,6 +95,42 @@
     }
 }
 
+LANGUAGES_PROVIDED_OBJ = {
+    "expected_type": dict,
+    "process": lambda s: s.process_languages_provided,
+    "allowed_nodes": ["*"],
+    "not_empty": True,
+    "child_nodes": {
+        "*": {
+            "expected_type": dict,
+            "required_nodes": ["name", "version", "apps"],
```

Comment:
I feel like `required_nodes`, `allowed_once_nodes` and setting `not_empty` in the `child_nodes` might not all be required. But I also do not know. Isn't a big deal either way just seems like it might not be required.